sky/clouds/cloud.py

```diff
@@ -439,5 +439,30 @@ def check_cluster_name_is_valid(cls, cluster_name: str) -> None:
                     'chars; maximum length is '
                     f'{max_cluster_name_len_limit} chars{cloud_name}.')
 
+    @classmethod
+    def check_disk_type_enabled(cls, instance_type: str,
+                                disk_type: str) -> None:
+        """Errors out if the disk type is not supported by the cloud provider.
+
+        Raises:
+            exceptions.NotSupportedError: If the disk type is not supported.
+        """
+        raise NotImplementedError
+
+    @classmethod
+    def get_disk_type(cls, disk_type: str) -> str:
+        """Returns the disk type name for each cloud."""
+        raise NotImplementedError
+
+    @classmethod
+    def get_disk_desc(cls, disk_type: str) -> str:
+        """Returns a string to describe the disk type for each cloud.
+
+        Default format: {disk_type}:{cloud_disk_type}
+
+        AWS format: {disk_type}:{cloud_disk_type}[{iops}]
+        """
+        return f'{disk_type}:{cls.get_disk_type(disk_type)}'
```

Comment:
Let's avoid showing this detailed information for now. Instead, we may want to explain the disk_type spec in our doc with the brief note about the performance: https://github.com/skypilot-org/skypilot/blob/master/docs/source/reference/yaml-spec.rst